src/common/cups_print.c

```diff
@@ -267,6 +279,10 @@ GList *dt_get_papers(const char *printer_name)
               paper->width = (double)size.width / 100.0;
               paper->height = (double)size.length / 100.0;
               result = g_list_append (result, paper);
+
+              dt_print(DT_DEBUG_PRINT,
+                       "[print] new med paper %4d %6.2f x %6.2f (%s) (%s)\n",
```

Comment:
Please replace `med` with some proper word that people can understand without looking at the code.